vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -498,6 +498,7 @@ def identify_request
   end
 
   def is_approver
+    # TODO: this should be current_user
```

Comment:
this should be `current_user.role_allows?`
But I think the specs are not setting up the session correctly and this is failing.

I'm leaving the values as is for now.
I will remove the TODO and put into an external todo task